src/frontend/src/components/common/List/index.tsx

```diff
@@ -36,7 +39,13 @@ const List = ({
           <thead>
             <tr>
               {columnsName.map((column, index) => (
-                <th key={index}>{column}</th>
+                <th key={index}>
+                  {column.split('"').map((name, idx) => (
+                    <div className={idx === 0 ? '' : 'no-bold'} key={name}>
+                      {name}
+                    </div>
+                  ))}
```

Comment:
I don't think splitting with string `"` is the right way to add anchor tags

Reply:
Refer Implementation here
src/frontend/src/components/public/PIAFormTabs/intake/helper/messages.ts
src/frontend/src/components/public/PIAFormTabs/intake/index.tsx

Reply:
<img width="998" alt="image" src="https://user-images.githubusercontent.com/87394256/222242283-65cdcdad-3e28-4945-99c3-d15e6f93124b.png">